sdk/communication/communication-administration/test/communicationIdentityClientWithToken.spec.ts

```diff
@@ -4,15 +4,15 @@
 import { assert } from "chai";
 import { Recorder } from "@azure/test-utils-recorder";
 import { CommunicationIdentityClient } from "../src";
-import { createRecordedCommunicationIdentityClientWithToken } from "./utils/recordedClient";
+import { createRecordedCommunicationIdentityClient } from "./utils/recordedClient";
 
 describe("CommunicationIdentityClientWithToken [Playback/Live]", function() {
   let recorder: Recorder;
   let client: CommunicationIdentityClient;
   let shouldSkip = false;
 
   beforeEach(function() {
-    const recordedClient = createRecordedCommunicationIdentityClientWithToken(this);
+    const recordedClient = createRecordedCommunicationIdentityClient(this, true);
```

Comment:
@0rland0Wats0n if we want to use the same function and only pass in a parameter. I will need make a change to this [PR](https://github.com/Azure/azure-sdk-for-js/pull/13271 ) for the `createRecordedPhoneNumberAdministrationClientWithToken` to stay consistent?

Reply:
Ah sorry, I forgot you were still working in that area. You can make the change if you have the time. Otherwise it's fine, it doesn't block your PR.